client/components/Overview/Overview components/ProductInfo components/Styles.jsx

```diff
@@ -1,15 +1,48 @@
 import React from 'react';
+import styled from 'styled-components';
 
+const StyledStyleThumbnail = styled.div`
+  width: 75px;
+  height: 75px;
+  background-color: yellow;
+  border-radius: 50%;
+`;
+
+const styles = {
+  tn: {
+    display: 'flex',
+    flexDirection: 'row'
+  }
+}
+
+//Props currently getting product info, not styles. Where does that info need to start?
```

Comment:
There is a specific api path to get the styles for a specific product. Its GET /products/:product_id/styles
If you go to the learn page on products api and scroll down you can see what the data looks like. I think that the get request should probably be done in the styles component since we probably dont need that information elsewhere.